lib/ModelMixins/TableMixin.ts

```diff
@@ -742,8 +742,10 @@ function TableMixin<T extends Constructor<Model<TableTraits>>>(Base: T) {
       return this.tableColumns.find(column => column.type === type);
     }
 
-    findColumnByName(name: string): TableColumn | undefined {
-      return this.tableColumns.find(column => column.name === name);
+    findColumnByName(name: string | undefined): TableColumn | undefined {
+      return isDefined(name)
+        ? this.tableColumns.find(column => column.name === name)
+        : undefined;
```

Comment:
I prefer allowing `undefined` argument, as most calls to `findColumnByName` use traits with optional values. It makes using optional chaining possible - otherwise you need to use if statement or ternary expressions before calling `findColumnByName`

For example

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/6187649/175466542-55b0103b-46a1-4cac-aa89-27bd799ba607.png">